### PR TITLE
build multi-platform hostplumber image

### PR DESCRIPTION
**Pull request by shweta50**

### CODE REVIEW

**Comment by cruizen** [comment]
Hi @shweta50 , is [ PMK-6478](https://platform9.atlassian.net/browse/PMK-6478) the JIRA issue associated with this - for the problem we are trying to solve? 

Have we seen this internally? Can you deploy hostplumber in a cluster and add testing notes?

**Review by poonam-agarwal28 — APPROVED** [review]

**Comment by cruizen** [comment]
Hi @shweta50 what releases do we need to backport this to? Can we identify the corresponding addon operator and Luigi versions? cc: @manasabsv26

**Review by vedantjoshi84 — APPROVED** [review]

**Comment by cruizen** [comment]
@Sanket36 , @vedantjoshi84 can we update this to push image to quay.io (to be used in kubevirt only initially - also the workflow execution should require approval by the committers like it did in Arlon) by setting Quay credentials as environment variables? Have we done that in any other GitHub actions?

**Pull request closed without merging**
